src/core/MREntity.js

```diff
@@ -307,13 +307,13 @@ export class MREntity extends MRElement {
      * @function
      * @description Callback function of MREntity - does nothing. Is called by the connectedCallback.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?